templates/partials/meta-tags.html

```diff
@@ -0,0 +1,58 @@
+{% set description = 
+  "OpenFEC is the Federal Election Commission's new app for exploring federal campaign finance data. Use OpenFEC to see how much money is raised and spent in federal elections. OpenFEC is currently in beta." 
+%}
+
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="description" content="{{ description }}">
+
+<!-- Mobile Specific Metas
+================================================== -->
+<meta name="HandheldFriendly" content="True">
+<meta name="MobileOptimized" content="320">
+<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
+
+<!-- Twitter
+================================================== -->
+<meta name="twitter:card" content="summary_large_image">
+<meta name="twitter:site" content="@18F">
+<meta name="twitter:creator" content="@18F">
+<meta name="twitter:title" content="{% if name %}{{ name }} - {% endif%}OpenFEC (Beta)">
+<meta name="twitter:description" content="{{ description }}">
+<meta name="twitter:image" content="/static/img/open-fec-tw.png">
+
+<!-- Facebook
+================================================== -->
+<meta property="og:type" content="website" />
+<meta property="og:url" content="https://open.fec.gov" />
```

Comment:
Do we want to hard code in the URL like this? Or should it be whatever URL of the page the person is on?